dbfit-java/informix/src/main/java/dbfit/environment/InformixFunctionStatementExecution.java

```diff
@@ -0,0 +1,50 @@
+package dbfit.environment;
+
+import java.sql.*;
+
+import dbfit.fixture.StatementExecution;
+
+public class InformixFunctionStatementExecution extends StatementExecution {
+    private Object returnValue = null;
+
+    public InformixFunctionStatementExecution(PreparedStatement statement, boolean clearParameters) {
+        super(statement, clearParameters);
+    }
+
+    @Override
+    public void run() throws SQLException {
+        ResultSet rs = statement.executeQuery();
+        try {
+            rs.next();
+            returnValue = rs.getObject(1);
+        } finally {
```

Comment:
In other places we've been using [try with resources syntax](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html). Might be appropriate here too?